### PR TITLE
added path support to code pipeline:

### PR DESCRIPTION
**Pull request by shapirov103**
*Issue #, if available:* #633  (potentially)

*Description of changes:*
Added support for path specification with code pipeline.  See docs for usage.

Customers can define their blueprints IaC code under arbitrary path in the repo. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/pipelines.md [hunk 1]
```diff
@@ -54,10 +54,22 @@ const blueprint = blueprints.EksBlueprint.builder()
...
+ blueprints.CodePipelineStack.builder()
+    .name("eks-blueprints-pipeline")
+    .repository({
+        owner: "aws-samples",
+        repoUrl: 'cdk-eks-blueprints-patterns',
+        credentialsSecretName: 'github-token',
+        path: "./infra/blueprints" // optional, default is './'
+    })```
+
 Note: the above code depends on the AWS secret `github-token` defined in the target account/region. The secret may be fined in one main region, and replicated to all target regions.
```
Comment (elamaran11) [inline docs/pipelines.md:73]: fined?

**Review by elamaran11 — CHANGES_REQUESTED** [review]
@shapirov103 Minor comment.

File: docs/pipelines.md (hunk 1 above)
Comment (shapirov103) [reply docs/pipelines.md]: addressed, good catch!

**Review by elamaran11 — APPROVED** [review]
LGTM

**Pull request merged**